app/services/api/v3/dashboards/parametrised_charts/flow_values_charts.rb

```diff
@@ -87,23 +95,23 @@ def single_year_no_ncont_charts
           def single_year_no_ncont_overview
             {
               source: :single_year_no_ncont_overview,
-              type: DYNAMIC_SENTENCE,
+              type: get_correct_chart_type(DYNAMIC_SENTENCE),
```

Comment:
in general let's not apply this logic to any "overview" charts, only "node type view" ones